README.md

```diff
@@ -46,6 +46,7 @@ module "full_vpc" {
 | dhcp\_options | DHCP options to assign to the VPC | <pre>object({<br>    domain_name          = string<br>    domain_name_servers  = list(string)<br>    netbios_name_servers = list(string)<br>    netbios_node_type    = number<br>    ntp_servers          = list(string)<br>  })</pre> | `null` | no |
 | ebs\_endpoint | Variables to provision an EBS endpoint to the VPC | <pre>object({<br>    private_dns_enabled = bool<br>    security_group_ids  = list(string)<br>    subnet_ids          = list(string)<br>  })</pre> | `null` | no |
 | ec2\_endpoint | Variables to provision an EC2 endpoint to the VPC | <pre>object({<br>    private_dns_enabled = bool<br>    security_group_ids  = list(string)<br>    subnet_ids          = list(string)<br>  })</pre> | `null` | no |
+| s3\_endpoint | Variables to provision an s3 endpoint to support new and existing subnets to the VPC | <pre>object({<br>  route_table_ids = list(string)<br>  })</pre> | `null` | no |
```

Comment:
This needs to be updated to match the variable

Reply:
done now i think :)